packages/pds/src/services/repo/index.ts

```diff
@@ -215,18 +215,20 @@ export class RepoService {
     commitData: CommitData,
     writes: PreparedWrite[],
   ) {
-    // @TODO move to appview
-    writes.map((write) => {
-      if (
-        write.action === WriteOpAction.Create ||
-        write.action === WriteOpAction.Update
-      ) {
-        this.labeler.processRecord(write.uri, write.record)
-      }
-    })
-
     const seqEvt = await sequencer.formatSeqCommit(did, commitData, writes)
     await sequencer.sequenceEvt(this.db, seqEvt)
+
+    // @TODO move to appview
+    this.db.onCommit(() => {
```

Comment:
oops this might overwrite notif processing & stuff

don't merge yet

Reply:
Oh dang!  Is it due to a possible bug in `onCommit()`?